src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -479,6 +478,9 @@ export function useDetailsValidation(): StepValidation {
   }, [blueprintId, name, setUniqueName, trigger, nameValid]);
 
   let nameError = '';
+  if (!name) {
```

Comment:
:+1: This is a great catch, nice job! Did we have an issue tracking this or did you just find it while working on the pull request? It seems like this bug has been here for a long time!

Reply:
I will open one - https://github.com/osbuild/image-builder-frontend/issues/2971